squirrel.go

```diff
@@ -61,27 +60,34 @@ type Runner interface {
 	QueryRower
 }
 
-type stdsql interface {
+// WrapStdSql wraps a type implementing the standard SQL interface with methods that
+// squirrel expects.
+func WrapStdSql(stdSql StdSql) Runner {
```

Comment:
Does this need to be exported? Who would use it?

Reply:
My intention here was this was to allow for use of stdlib dbs with the `QueryRowWith` function. Eg:

```go
err := sq.QueryRowWith(WrapStdSql(myCustomWrapper), sql.Select("foo")).Scan(...)
```

Would it be preferable to cut this down to something like the following?

 ```go
type stdQueryRower interface {
  QueryRow(string, ...interface{}) *sql.Row
}

func WrapQueryRower(queryRower stdQueryRower) QueryRower
```

Reply:
Hey, just like your initial PR comment says! If only I'd read that more closely...